### PR TITLE
Adds the possibility of having models with no centroids in the project file

### PR DESCRIPTION
**Pull request by pedrocamargo**
Adds the possibility of having models with no centroids in the project file
Refactoring of the graph class means that this can be used without the project as well

### CODE REVIEW

**Comment by pedrocamargo** [comment]
Addresses issue #128

File: aequilibrae/project/network/network.py [hunk 1]
```diff
@@ -276,7 +276,7 @@ def build_graphs(self) -> None:
             warn(f'Fields were removed form Graph for being non-numeric: {",".join(removed_fields)}')
 
         curr.execute('select node_id from nodes where is_centroid=1;')
-        centroids = np.array([i[0] for i in curr.fetchall()])
+        centroids = np.array([i[0] for i in curr.fetchall()], np.int32)
```
Comment (janzill) [inline aequilibrae/project/network/network.py:279]: type here is different from graph.py, is that on purpose?

Comment (pedrocamargo) [reply aequilibrae/project/network/network.py:279]: Not on purpose. Good catch

**Comment by pedrocamargo** [comment]
Thanks for the review, @janzill !!

**Pull request merged**